pkg/rego/load.go

```diff
@@ -1,16 +1,18 @@
 package rego
 
 import (
+	"fmt"
 	"io/fs"
 	"os"
 	"path/filepath"
 
+	"github.com/fugue/regula/pkg/loader"
 	embedded "github.com/fugue/regula/rego"
 	"github.com/open-policy-agent/opa/ast"
 	"github.com/open-policy-agent/opa/rego"
 )
 
-var loadExts map[string]bool = map[string]bool{
+var opaExts map[string]bool = map[string]bool{
```

Comment:
This is something I'm less familiar with. I've been using this pattern for set functionality in a few places and my understanding is that it works well because the zero value for `bool` is `false`. So if I try to get a member of the map that doesn't exist, I get `false` which allows statements like `if opaExts[ext]` to work without needing to do the longer `if _, ok := opaExts[ext]; ok`. Would the same thing work with `map[string]struct{}`?

Reply:
Using `struct{}` is common but I prefer using `bool` myself since it's easier to type and the default value of false is logical IMO.